v2/index.html

```diff
@@ -965,27 +965,27 @@ <h4>Accidentals</h4>
             <h4>Rests</h4>
             <div class="issue" data-number="29"></div>
             <div class="issue" data-number="41"></div>
-            <p>Rests for single notes are indicated by <code>-</code> (a minus sign).</p>
-            <table class="simple">
-                <thead>
-                <tr>
-                    <th>Code</th>
-                    <th>Name</th>
-                </tr>
-                </thead>
-                <tbody>
-                <tr>
-                    <td><code>-</code></td>
-                    <td>single-note rest (preceded by rhythmic value like note names)</td>
-                </tr>
-                <tr>
-                    <td><code>=</code></td>
-                    <td>measure rest (followed by number of measures and a bar line)</td>
-                </tr>
-
-                </tbody>
-            </table>
-
+            <p>
+                Rests for single notes MUST be indicated by a hyphen/minus character <code>-</code>. This character
+                MUST be preceded by a <a>duration</a> value giving the musical duration of the rest.
```

Comment:
This is not really true. A rest will just use the current duration if not duration is indicated

Reply:
OK, I can change this.